README.rst

```diff
@@ -94,7 +94,7 @@ root directory of oh-my-zsh:
 
 ::
 
-    git clone https://github.com/MichaelAquilina/zsh-you-should-use.git $ZSH/custom/plugins/zsh-you-should-use
+    git clone https://github.com/MichaelAquilina/zsh-you-should-use.git $ZSH/custom/plugins/you-should-use
```

Comment:
not sure I agree with this. While the project title on the README is `You Should Use`, the repo name is still `zsh-you-should-use` which will make it easier to identify in case of conflicts. Is there any particular reason you are proposing this change?

Reply:
Ah I see what you mean. Does the name of the plugin and the directory needs to be the same? 

I don't use zsh but if this is the case then this seems like a sensible change.